physics/satmedmfvdifq.meta

```diff
@@ -596,6 +596,43 @@
   type = real
   kind = kind_phys
   intent = in
+#IVAI
+[claie]
+  standard_name = leaf_area_index_eccc
```

Comment:
Its critical that these standard names match the exactly the standard names in the fv3atm/ccpp/data/GFS_typedefs.meta file.  I make suggestions here and other variables to match what you have in fv3atm side.
```suggestion
  standard_name = canopy_leaf_area_index
```